src/components/TodoList.jsx

```diff
@@ -41,6 +42,14 @@ export function TodoList() {
     setTodoName("");
     setDueDate("");
   };
+  //タスクの削除
+  const deleteTodo = (targetId) => {
+    const newTodos = todos.filter((todo) => todo.id !== targetId);
+    // IDの番号を振り直す
+    const newId = newTodos.map((todo, index) => ({ ...todo, id: index + 1 }));
```

Comment:
`newId`という命名だと「新しい1つのID」という意味に受け取られてしまいます。
僕だったらこう実装します

```suggestion
    const newTodos = todos.filter((todo) => todo.id !== targetId).map((todo, index) => ({ ...todo, id: index + 1 }));;
```

Reply:
newIdと言う命名は不適切、了解しました！
確かにそう言う意味になります。
const newTodos = todos.filter((todo) => todo.id !== targetId).map((todo, index) => ({ ...todo, id: index + 1 }));;
こういう書き方もあるのですね！ありがとうございます！直します！